src/targets/gpu/lowering.cpp

```diff
@@ -236,11 +240,37 @@ struct miopen_apply
             {
                 check_shape(s, insert_precompile_op(it));
             }
+            else if(attrs.contains("custom_op"))
```

Comment:
This should be `attr.contains("target")`.

Reply:
Done